### ISSUE TITLE
Create Index with Column Collate Not supported Yet

### ISSUE THREAD

**Issue by dodilp** [yugabyte/yugabyte-db#1480]
<img width="284" alt="Screen Shot 2019-06-04 at 8 35 18 AM" src="https://user-images.githubusercontent.com/2207029/58879476-bff7f380-86a3-11e9-8d29-083dd149cd29.png">

**Comment by rkarthik007** [comment]
cc @m-iancu for more details.

@dodilp - you seem to be using an app/tool against YugaByte DB based on the UI screenshot, curious what it is?

**Comment by ndeodhar** [comment]
Thanks for reporting this, @dodilp . This is a subset of https://github.com/YugaByte/yugabyte-db/issues/1127. We'll most likely support collation after our 2.0 release in Aug. In this particular case, since you are using default collation, you can drop the collate clause from your create index statement.

**Comment by m-iancu** [comment]
@dodilp 
We may be able to support that option _for the default case only_ in the near future.
Otherwise, as @ndeodhar mentioned, we will not support collation in general before Aug.
Could you let us know which tool you are using (looks like it might be `pgadmin` based on the screenshot) and if possible how to replicate the issue in that tool -- then it's easier for us to test.

Meanwhile, for testing, ideally you can edit the query to remove that option.
If that is not possible (e.g. because it is generated by a tool) we have a workaround to suppress such feature-unsupported errors:
```
export YB_SUPPRESS_UNSUPPORTED_ERROR=1
./bin/yb-ctl stop && ./bin/yb-ctl start
```
Now that query should return a `WARNING` and proceed instead of throwing an error.
_Note: this flag is not safe in general/prod as some options may be ignored or misbehave, but can be useful for initial testing/compatibility._

**Comment by dodilp** [comment]
Yes, the tool is PgAdmin4. The script can be auto-generated using create index dialog
<img width="530" alt="Screen Shot 2019-06-04 at 11 14 46 PM" src="https://user-images.githubusercontent.com/2207029/58927941-8c0de400-871e-11e9-9360-c893f21a187f.png">

**Comment by m-iancu** [comment]
@dodilp Filed a generic issue (#1497) for this that we will look at.
In the meantime, either using the `YB_SUPPRESS_UNSUPPORTED_ERROR` flag as described above, or simply creating the objects (e.g. index) through direct SQL commands should work.

**Comment by kriteshranjan** [comment]
Closing this issue based on AI validation. cc: @m-iancu , @VamsidharGunturu
